src/Storage/appsettings.json

```diff
@@ -32,5 +32,9 @@
   },
   "PepSettings": {
     "PdpDecisionCachingTimeout": 5
+  },
+  "FileScanQueueSettings": {
+    "FileScanQueueName": "file-scan-inbound",
+    "ConnectionString": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1"
```

Comment:
I don't think we have a task for it, but this connection string should be stored in KV during generation of the storage account. Perhaps Consider agreeing on naming of the property with @tba76 so the secret is only stored once, but used by both storage and the function? given that we are referencing the same resource

Reply:
Connection string is persisted in key vault during provisioning as `QueueStorageSettings--ConnectionString`

Reply:
My bad. The function secret is defined seperate of what is loaded into key vault, so duplicating settings name from the function won't work. We have two options here
1) Name it `QueueStorageSettings` to match what is is KV today for this storage account
2) Update tf to duplicate the secret with the desired name `FileScanQueueSettings` in KV during provisioning.

Thinking of it, maybe a 3rd as well.. Read the QueueStorageSettings into the `FileScanQueueSettings` class when setting up the configuration in Program.cs. Although this would make it hard to follow the code and find the settings use in kv at a later point I guess

Reply:
To clarify, the issue isn't the queue name, but the if for the settings section `FileScanQueueSettings` which we will need to override with a connection from kv.

Reply:
I would have preferred option 2, but have now implemented 1.